sqlite/lib/SQLiteService.js

```diff
@@ -44,13 +44,75 @@ class SQLiteService extends SQLService {
 
   prepare(sql) {
     try {
-      return this.dbc.prepare(sql)
+      const stmt = this.dbc.prepare(sql)
+      stmt._run = stmt.run
+      stmt.run = this._run.bind(stmt)
+      stmt._iterator = this._iterator
+      stmt.stream = this._stream.bind(stmt)
+      return stmt
```

Comment:
Looks like an ugly workaround for something... → for what?

Reply:
This is to support streams as parameters to the `run` function which SQLite doesn't support natively.